pyanaconda/ui/gui/spokes/installation_source.py

```diff
@@ -618,12 +621,18 @@ def changed(self):
         update_payload_repos = self._update_payload_repos()
         return method_changed or update_payload_repos or self._error
 
+    @property
+    def CDN_used_as_installation_source(self):
```

Comment:
Please, use only lowercase letters.

Reply:
This property looks like a hack to me.

Reply:
Still seems less problematic to me than touching the method based logic in any way. I'll ad a note to look at this as a followup, but I'm not sure if we have better options with the state of the Source spoke code on RHEL 8.

Reply:
Switched to lowercase.